app/views/docs/authentication.phtml

```diff
@@ -65,60 +125,149 @@ promise.then(function (response) {
+
+val user = account.createAnonymousSession()</code></pre>
+        </div>
+    </li>
+</ul>
 
 <h3><a href="/docs/authentication#oauth" id="oauth">OAuth</a></h3>
 
 <p>
-    OAuth is another way to authenticate a user using a multi-step process. When using OAuth to authenticate, the authentication request is initiated from the client application. The user is then redirected to an OAuth2 provider to complete the authentication step, and finally, the user is redirected back to the client application. This provides integration with many third-party services that provide their own OAuth integration as a more secure approach than providing a username/password directly.
+    OAuth is another way to authenticate a user using a multistep process. When using OAuth to authenticate, the authentication request is initiated from the client application. The user is then redirected to an OAuth2 provider to complete the authentication step, and finally, the user is redirected back to the client application. This provides integration with many third-party services that provide their own OAuth integration as a more secure approach than providing a username/password directly.
 </p>
 
 <p>
     In applications with first-party redirects, using OAuth2 for authentication is preferred. 
 </p>
 
 <p>
-    The example below shows you how to authenticate with OAuth2 using the Web SDK and Amazon's OAuth system.
+    The example below shows you how to authenticate with OAuth2 using Amazon's OAuth system.
 </p>
 
-<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
-    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+        <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
 
 const client = new Client();
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2');               // Your project ID
 
 const account = new Account(client);
 
-client
+// Go to OAuth provider login page
+account.createOAuth2Session('amazon');</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+
+final client = Client()
     .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
-    .setProject('5df5acd0d48c2') // Your project ID
-;
+    .setProject('5df5acd0d48c2');               // Your project ID
+
+final account = Account(client);
 
 // Go to OAuth provider login page
-account.createOAuth2Session('amazon');</code></pre>
-</div>
+await account.createOAuth2Session('amazon');</code></pre>
```

Comment:
provider in dart/flutter is a named parameter `provider: 'amazon'`. Please make sure you check for android as well.